src/util/ProtocolVersion.h

```diff
@@ -33,7 +33,8 @@ enum class ProtocolVersion : uint32_t
     V_18,
     V_19,
     V_20,
-    V_21
+    V_21,
+    V_22
```

Comment:
Don't forget to update this to include v23 and use it where appropriate.
I also would suggest using a named constant instead of the direct version checks in order to make changes easier.